src/applications/check-in/utils/defineWebComponents.js

```diff
@@ -0,0 +1,18 @@
+import '@department-of-veterans-affairs/component-library/dist/main.css';
+import {
+  defineCustomElementVaAccordion,
+  defineCustomElementVaAccordionItem,
+  defineCustomElementVaAlert,
+  defineCustomElementVaLoadingIndicator,
+  defineCustomElementVaMemorableDate,
+  defineCustomElementVaTelephone,
+  defineCustomElementVaTextInput,
+} from '@department-of-veterans-affairs/component-library/dist/components';
+
+defineCustomElementVaAccordion();
+defineCustomElementVaAccordionItem();
+defineCustomElementVaAlert();
+defineCustomElementVaLoadingIndicator();
+defineCustomElementVaMemorableDate();
+defineCustomElementVaTelephone();
+defineCustomElementVaTextInput();
```

Comment:
This is essentially defining a subset of the Web Components that get defined in this file:

https://github.com/department-of-veterans-affairs/vets-website/blob/f08c3a24a4bc4bc65232c5e5e668fc8c5a9c335b/src/platform/site-wide/wc-loader.js#L2-L10

Another key difference is that with the individual component definitions, we don't get any of Stencil's lazy loading.